crates/holochain_types/src/app.rs

```diff
@@ -200,7 +200,7 @@ pub type RoleSettingsMap = HashMap<RoleName, RoleSettings>;
 
 /// Settings for a Role that may be passed on installation of an app
 #[derive(Debug, serde::Serialize, serde::Deserialize)]
-#[serde(tag = "type")]
+#[serde(tag = "type", rename_all = "snake_case")]
```

Comment:
I understood your reasoning for not tagging the content here. But now that I see all the test together, this one stands out with being the only untagged enum and if we're striving for consistency, it feels like we're missing the mark. I vote for tagging the content here.